src/sentry/db/models/fields/guid.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from django.conf import settings
+from django.db.models import CharField
+from uuid import uuid4
+
+
+def generate_guid():
+    return uuid4().hex
+
+
+class GUIDField(CharField):
```

Comment:
since we're adding this new, can we store this as actual guid in the db if possible and fall back to a hex string without dashes? We already went into some issues in the past where case sensitivity was a problem with these types of guids.

Reply:
will try to import uuidfield from django-pg